docker/api/container.py

```diff
@@ -520,6 +520,10 @@ def create_host_config(self, *args, **kwargs):
                 - ``container:<name|id>`` Reuse another container's network
                   stack.
                 - ``host`` Use the host network stack.
+                  This mode is incompatible with ``port_bindings``.
+                  If ``host`` is used as network_mode, all of listed up to
```

Comment:
port bindings are not ignored in this case, the 2 options are not compatible (raising an error)

Reply:
Same in following doc updates

Reply:
Yes, right. the ```host``` mode is not abled to use with ```port_bindings```. I will update docs.